### DISCUSSION TITLE
The UniGetUI tool doesn't work, it doesn't detect any software. WinGetUI worked ...

### DISCUSSION THREAD

**Discussion by sypqys** [Devolutions/UniGetUI#2796] [Q&A]
hello

it's all in the title
I use Hard_Configurator and Portmaster. After that, I don't know what's wrong...

![UniGetUI_j4KjlyMZCq](https://github.com/user-attachments/assets/c66b5bb0-bc17-4c73-b56e-0cc6743727eb)

Thanks

**Comment by marticliment** [comment]
Go to Settings -> WinGet -> Enable built-in WinGet instead of COM API

**Reply by sypqys** [reply]
This is the same with that on my computer.

**Reply by marticliment** [reply]
I am working on a troubleshooter for this. Please wait for the next update

**Comment by sypqys** [comment]
It seems to work now, but it launches this on startup.

![UniGetUI_GRgYir8ekQ](https://github.com/user-attachments/assets/20ebca25-5816-4b97-a874-9a4c4476d7ad)
